When testing `rhasheq `usage, the following error occurs.

```
Error		CMake Error at F:/c89stringutils/installed/x64-windows/share/rhasheq/rhasheqTargets.cmake:60 (set_target_properties):
  The link interface of target "rhasheq" contains:

    LibRHash::LibRHash

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.		F:/c89stringutils/installed/x64-windows/share/rhasheq/rhasheqTargets.cmake	60	
```
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

--- a/ports/rhasheq/find-rhash.patch
+++ b/ports/rhasheq/find-rhash.patch
@@ -1,0 +1,26 @@
+diff --git a/rhasheq/CMakeLists.txt b/rhasheq/CMakeLists.txt
+index fb7357f..baaaaf8 100644
+--- a/rhasheq/CMakeLists.txt
++++ b/rhasheq/CMakeLists.txt
+@@ -23,8 +23,9 @@ else ()
+ endif (VCPKG_TOOLCHAIN)
+ 
+ set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules;${CMAKE_MODULE_PATH}")
+-find_package(LibRHash)
++find_package(unofficial-rhash CONFIG REQUIRED)
+ 
++if(0)
+ if (NOT LibRHash_FOUND)
+     set(CMAKE_FIND_LIBRARY_PREFIXES ${CMAKE_FIND_LIBRARY_PREFIXES} lib)
+     set(CMAKE_FIND_LIBRARY_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES} _imp.lib -d.lib -d_imp.lib)
+@@ -40,8 +41,9 @@ endif (NOT TARGET LibRHash)
+ if (NOT LibRHash_FOUND)
+     message(FATAL_ERROR "LibRHash not found")
+ endif (NOT LibRHash_FOUND)
++endif()
+ 
+-target_link_libraries("${LIBRARY_NAME}" INTERFACE "LibRHash::LibRHash")
++target_link_libraries("${LIBRARY_NAME}" INTERFACE unofficial::rhash::rhash)
+ 
+ include(GNUInstallDirs)
+ target_include_directories(

--- a/ports/rhasheq/find-rhash.patch
+++ b/ports/rhasheq/find-rhash.patch
@@ -1,3 +1,16 @@
+diff --git a/cmake/Config.cmake.in b/cmake/Config.cmake.in
+index c64c475..d57179e 100644
+--- a/cmake/Config.cmake.in
++++ b/cmake/Config.cmake.in
+@@ -1,4 +1,8 @@
+ 
+ @PACKAGE_INIT@
+ 
++include(CMakeFindDependencyMacro)
++
++find_dependency(unofficial-rhash)
++
+ include ( "${CMAKE_CURRENT_LIST_DIR}/rhasheqTargets.cmake" )
 diff --git a/rhasheq/CMakeLists.txt b/rhasheq/CMakeLists.txt
 index fb7357f..baaaaf8 100644
 --- a/rhasheq/CMakeLists.txt

--- a/ports/rhasheq/portfile.cmake
+++ b/ports/rhasheq/portfile.cmake
@@ -4,6 +4,8 @@ vcpkg_from_github(
     REF cf5442f6468871beb6088991501e6ba052fe4467
     SHA512 2be63bc32c7cea35eefbcc7186255e9cd8ba6ed4b03cb9ee2e62740fe17ad74a81ee7e4ada37f4a2dca99a82becf790d6aa86822cdbeba3972cf11d54095ca91
     HEAD_REF master
+    PATCHES
+        find-rhash.patch
 )
 
 vcpkg_cmake_configure(

--- a/versions/r-/rhasheq.json
+++ b/versions/r-/rhasheq.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "255cfa293a9a2a64778ccfb88a5e21f2f4c063a9",
+      "git-tree": "9b9d7eec44e958e70a4d732c7d815b13995c93d6",
       "version-date": "2023-06-17",
       "port-version": 0
     },

